ai-ml-samples/interactive/ImageClassificationInSpark.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<p>Can you split this up into three separate code blocks and provide more of an explanation as to what each is doing? I can help if you'd like.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudDataproc/cloud-dataproc/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Done</p>